### PR TITLE
IBX-7489: `Change the password` section added

### PR DESCRIPTION
**Pull request by julitafalcondusza**
| Question      | Answer
| ------------- | ---
| JIRA Ticket   | (https://github.com/ibexa/admin-ui/pull/1095)
| Versions      | master

`Change the password` section added in "Getting started" page.

#### Checklist

- [x] Text renders correctly
- [x] Text has been checked with vale
- [x] Description metadata is up to date

### CODE REVIEW

File: docs/getting_started/get_started.md [hunk 1]
```diff
@@ -132,3 +132,15 @@ such as preferred timezone, short and full date and time format, or Back Office
...
+### Change the password
+
+You can change your user password at any time.
+To do it, first, access your user profile, and go to **Account settings** tab.
+Then, click **Change password**.
+
+![Change password](img/change_password.png "Change password")
+
+Fill out all the required fields and click **Save and close** to save changes.
+Click **Discard** to remove your changes and return to the previous screen.
```
Comment (juskora) [inline docs/getting_started/get_started.md:144]:
```suggestion
Click **Discard** to reject changes and return to the previous screen.
```

File: docs/getting_started/get_started.md [hunk 2]
```diff
@@ -132,3 +132,15 @@ such as preferred timezone, short and full date and time format, or Back Office
...
 
+### Change the password
+
+You can change your user password at any time.
+To do it, first, access your user profile, and go to **Account settings** tab.
+Then, click **Change password**.
+
+![Change password](img/change_password.png "Change password")
+
+Fill out all the required fields and click **Save and close** to save changes.
```
Comment (juskora) [inline docs/getting_started/get_started.md:143]:
```suggestion
Fill in all the required fields and click **Save and close** to save changes.
```

**Review by juskora — CHANGES_REQUESTED** [review]
I would mention here a link to password policy section in the docs :)

**Review by juskora — APPROVED** [review]

File: docs/getting_started/get_started.md [hunk 3]
```diff
@@ -132,3 +132,15 @@ such as preferred timezone, short and full date and time format, or Back Office
 --------|-----------|
 |Active dashboard|Sets which dashboard is displayed after you log in.|
 
+### Change the password
+
+You can change your user [password]([[= developer_doc =]]/users/passwords/) at any time.
```
Comment (dabrt) [inline docs/getting_started/get_started.md:137]:
```suggestion
You can change your user [password]([[= developer_doc =]]/users/passwords/) at any time.
```

I would add a disclaimer here: "If the administrator granted you the right permissions, ..."

**Review by dabrt — APPROVED** [review]

File: docs/getting_started/get_started.md (hunk 3 above)
Comment (julitafalcondusza) [reply docs/getting_started/get_started.md:137]: It's explained in the "Passwords" page mentioned in link.

**Pull request merged**